src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -1138,44 +1136,7 @@ subroutine decomp_rates_mimics(bounds, num_soilc, filter_soilc, &
                pf = p - col%patchi(c)
                s  = clm_fates%f2hmap(nc)%hsites(c)
                annsum_npp(p) = clm_fates%fates(nc)%bc_out(s)%annsum_npp_pa(pf)
-
-               ! Values by pft from Table 1 in Brovkin et al. (2012):
-               ! https://bg.copernicus.org/articles/9/565/2012/bg-9-565-2012.pdf
-               ! Not moving to the params files at this time: Hardwiring will
-               ! become unnec. when we calc. ligninNratioAvg the same way for
-               ! FATES and non FATES cases.
-               select case (pftname(patch%itype(p)))
-                  case ('not_vegetated')
-                     ligninNratio(p) = 0._r8
-                  case ('broadleaf_evergreen_tropical_tree')
-                     ligninNratio(p) = 18.7_r8  ! 17.8 / 0.95
-                  case ('broadleaf_deciduous_tropical_tree')
-                     ligninNratio(p) = 8.9_r8  ! 14.5 / 1.63
-                  case ('needleleaf_evergreen_temperate_tree')
-                     ligninNratio(p) = 33.4_r8  ! 24.4 / 0.73
-                  case ('broadleaf_evergreen_temperate_tree', &
-                        'broadleaf_evergreen_shrub')
-                     ligninNratio(p) = 24.4_r8  ! 21.0 / 0.86
-                  case ('broadleaf_deciduous_temperate_tree', &
-                        'broadleaf_deciduous_temperate_shrub')
-                     ligninNratio(p) = 16.6_r8  ! 16.9 / 1.02
-                  case ('needleleaf_evergreen_boreal_tree', &
-                        'needleleaf_deciduous_boreal_tree')
-                     ligninNratio(p) = 26.7_r8  ! 25.6 / 0.96
-                  case ('broadleaf_deciduous_boreal_tree', &
-                        'broadleaf_deciduous_boreal_shrub')
-                     ligninNratio(p) = 25.1_r8  ! 22.3 / 0.89
-                  case ('c4_grass', 'miscanthus', 'irrigated_miscanthus', &
-                        'switchgrass', 'irrigated_switchgrass', &
-                        'tropical_corn', 'irrigated_tropical_corn', &
-                        'temperate_corn', 'irrigated_temperate_corn', &
-                        'sugarcane', 'irrigated_sugarcane', &
-                        'sorghum', 'irrigated_sorghum', &
-                        'millet', 'irrigated_millet')  ! all c4
-                     ligninNratio(p) = 24.1_r8  ! 23.4 / 0.97
-                  case default  ! all pfts/cfts that do not fit above categories
-                     ligninNratio(p) = 12.1_r8  ! 16.6 / 1.37
-               end select
+               ligninNratio(p) = clm_fates%fates(nc)%bc_out(s)%litt_flux_ligc_per_n(pf)
```

Comment:
@rgknox @ekluzek 
This is the change I made so as to use the new variable `litt_flux_ligc_per_n` calculated by FATES. I followed the template of what I had done with the other FATES variable used here `annsum_npp_pa`.

**Two problems that I need to go over with you:**
1) I had to make some modifications to FATES for the code to build.
2) The test that I introduced in this PR runs, but COMPARE_base_rest fails.

**The details:**
1) I added 1 line and changed 2 lines in FATES for the build to complete:
`FatesInterfaceMod.F90`
@@ -305,8 +305,8 @@ contains
```
     fates%bc_out(s)%rootr_pasl(:,:) = 0.0_r8
     fates%bc_out(s)%btran_pa(:)     = 0.0_r8

-    ! MIMIC litter quality, always initialize to unset
-    fates%bc_out(s)%litt_flux_ligc_per_n = fates_unset_r8
+    ! MIMICS litter quality, always initialize to unset
+    fates%bc_out(s)%litt_flux_ligc_per_n(:) = fates_unset_r8
```
@@ -589,6 +589,8 @@ contains
```
       allocate(bc_out%ftid_parb(maxPatchesPerSite,hlm_numSWb))
       allocate(bc_out%ftii_parb(maxPatchesPerSite,hlm_numSWb))

+      ! MIMICS
+      allocate(bc_out%litt_flux_ligc_per_n(maxPatchesPerSite))
```

`FatesInterfaceTypesMod.F90`
@@ -609,8 +609,7 @@ module FatesInterfaceTypesMod
```
       ! MIMICS Boundary Conditions
       ! -----------------------------------------------------------------------------------
-      real(r8) :: litt_flux_ligc_per_n  ! lignin carbon per total nitrogen
-                                        ! in the fragmentation flux, per square meter [g/g]
+      real(r8), allocatable :: litt_flux_ligc_per_n(:)  ! lignin carbon per total nitrogen in the fragmentation flux, per square meter [g/g]
```

2) Regarding the failing restart, I wanted to check in before trying to solve this:
@rgknox I suspect that some part of the calculation leading to the new variable `litt_flux_ligc_per_n` must go to restart. I will start looking, but I wanted your input, in case your familiarity with the calculation leads you to a quick answer.

Reply:
@rgknox looking at the code more:
I started with the assumption of a patch-level variable, but I think you're calculating `litt_flux_ligc_per_n` at the Site level. If so, I need to revisit my changes and see it I get things to work without additional mods on the FATES side.

Reply:
That was the problem. The test now passes.

Feel free to review my mods for further comments/suggestions/corrections.

Meanwhile I will kick off the test suites.

Reply:
ok, great, nevermind, I see you did exactly this in commit dd24018

Reply:
Perfect, thank you for that confirmation, @rgknox .

Good news with the test suites:
Cheyenne PASS
Izumi OK (i.e. 2 expected failures)